exporter/containerimage/writer.go

```diff
@@ -551,11 +551,10 @@ func (ic *ImageWriter) Applier() diff.Applier {
 func defaultImageConfig() ([]byte, error) {
 	pl := platforms.Normalize(platforms.DefaultSpec())
 
-	img := ocispecs.Image{
-		Architecture: pl.Architecture,
-		OS:           pl.OS,
-		Variant:      pl.Variant,
-	}
+	img := ocispecs.Image{}
+	img.Architecture = pl.Architecture
+	img.OS = pl.OS
+	img.Variant = pl.Variant
```

Comment:
This is where I noticed that in some cases we're discarding the `OSVersion` and `OSFeatures` fields (there may be other locations where we do this). Perhaps this is intentional, but we should probably review the code to see if we _unintentionally_ discard those fields, and update the code where needed.